cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -409,14 +409,14 @@ users:
 					AlwaysAllowPaths:             []string{"/healthz"}, // note: this does not match /healthz/ or /healthz/*
 				},
 			},
-			expectedUsername: "none, http",
+			expectedUsername: []string{"none, http"},
```

Comment:
Does this need to be "none, http" instead of "none" or "http"?

Reply:
Please check the mock codes and the  "none, http" is hard-coded here.

insecureserver := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
		username, _, _ = req.BasicAuth()
		if username == "" {
			username = "none, http"
		}
		w.WriteHeader(200)
		w.Write([]byte(`ok`))
	}))

Reply:
Correct, I asked the question because reviously you changed it to []string{"none", "http"}.

Reply:
I checked the history and there is no []string{"none", "http"} change.